app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  /**
+   * Widget for entering PVC information
+   *
+   * model:
+   *   The model for the input. The model will either use or add the following keys:
+   *     {
+   *       name: "",
+   *       accessMode: "",
+   *       Capacity: ""
+   *     }
+   *
+   */
+  .directive("oscPersistentVolumeClaim", function(){
+    return {
+      require: '^form',
```

Comment:
Since you use `ng-form` in the view, I don't think you need this. You can probably remove `require: '^form'` as well as the `controller` and `link` functions below.